docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -107,20 +75,35 @@ spec:
       vlan: 22
 EOF
 ----
++
+* In case when `neutron-sriov-nic-agent` is running on the existing Compute nodes, physical device mappings needs to be checked and set the same in the
+`OpenStackDataPlaneNodeSet` custom resource (CR). Those options will need to be set in the `OpenStackDataPlaneNodeSet` CR. For more information, see xref:reviewing-the-openstack-control-plane-configuration_adopt-control-plane[Reviewing the {rhos_prev_long} control plane configuration].
+
+.Variables
+
+Define the shell variables used in the Fast-forward upgrade steps below.
+Set `FIP` to the floating IP address of the `test` VM pre-created earlier on the source cloud.
+Define the map of Compute node name, IP pairs.
+The values are just illustrative, use values that are correct for your environment:
+
+----
+PODIFIED_DB_ROOT_PASSWORD=$(oc get -o json secret/osp-secret | jq -r .data.DbRootPassword | base64 -d)
 
-* In case when neutron-sriov-nic-agent is running on the existing compute nodes,
-  physical device mappings needs to be checked and set the same in the
-  OpenStackDataPlaneNodeSet CR. Those options will need to be set in the
-  OpenStackDataPlaneNodeSet CR.
-  For more information, see xref:pulling-the-openstack-configuration_{context}[Pulling the OpenStack configuration].
+alias openstack="oc exec -t openstackclient -- openstack"
+FIP=192.168.122.20
+declare -A computes
+export computes=(
+  ["standalone.localdomain"]="192.168.122.100"
+  # ...
+)
+----
 
-== Adopting the data plane
+.Procedure
 
 * _Temporary fix_ until the OSP 17 https://code.engineering.redhat.com/gerrit/q/topic:stable-compute-uuid[backport of the stable compute UUID feature]
-lands.
-+
-For each compute node grab the UUID of the compute service and write it too
-the stable `compute_id` file in `/var/lib/nova/` directory.
+lands. 
+//kgilliga: Can this text be removed? I think the fix was merged?
```

Comment:
@gibizer ^^ I think we will have to keep it until we have a fresh openstack containers build released, which would contain the needed stable id fixes?

Reply:
If we know that all the 17 clouds we are adoption are running 17.1.3 or later then this procedure should be removed from the doc procedure.
There is the same logic in the CI ansible tasks in this repo. Those can be removed only after 17.1.3 is on the CDN and the CI jobs are using that to deploy the 17 cloud during the testing.

Reply:
I will leave this text until 17.1.3 is on the CDN, and then we can revisit this.